src/components/AddForm.jsx

```diff
@@ -1,31 +1,32 @@
 import { FcPlus } from 'react-icons/fc';
-import './AddForm.css'
+import './AddForm.css';
 
-const InputBook = () => {
-
-    return (
-      <>
-        <form className="form-container">
-          <input
-            type="text"
-            placeholder="Add Author"
-            className="input-text"
+function AddForm() {
+  return (
+    <>
+      <form className="form-container">
+        <input type="text" placeholder="Add Author" className="input-text" />
+        <button className="input-submit" type="button">
+          <FcPlus
+            style={{
+              fontSize: '20px',
+              marginTop: '2px',
+            }}
           />
-           <input
-            type="text"
-            placeholder="Add Title"
-            className="input-text"
+        </button>
+        <br />
+        <br />
+        <input type="text" placeholder="Add Title" className="input-text" />
+        <button className="input-submit" type="button">
+          <FcPlus
+            style={{
+              fontSize: '20px',
+              marginTop: '2px',
+            }}
           />
-          <button className="input-submit" type="button">
-            <FcPlus
-              style={{
-                fontSize: '20px',
-                marginTop: '2px',
-              }}
-            />
-          </button>
-        </form>
-      </>
-    );
-  };
-  export default InputBook;
\ No newline at end of file
+        </button>
+      </form>
+    </>
```

Comment:
- [ ] Kindly, for the ```addform```, I think it would be better to use a single submit button, on the add book form, 😀 Having a **single submit button**, provides consistency to the user experience. 
Users expect to see one button that they can click to submit their form data, and having multiple submit buttons can confuse them. 😎

![image](https://user-images.githubusercontent.com/102757126/221227414-0fbec7fa-d19c-4014-a94f-c0c3b96b93e3.png)